apps/client/src/components/Location/LocationInput.tsx

```diff
@@ -61,6 +71,14 @@ const LocationInput: React.FC<{
         action: actions.SUBMIT_LOCATION,
         name: address.postalCode.substring(0, 4),
       });
+      matomoTrackEvent({
+        action: actions.SUBMIT_LOCATION,
+        name: monument ? monument : eventNames.NO_MONUMENT,
+      });
+      matomoTrackEvent({
+        action: actions.SUBMIT_LOCATION,
+        name: cityScape ? cityScape : eventNames.NO_CITYSCAPE,
+      });
```

Comment:
Do we really need to send 3 separate events?
Is there no way to send it as 1 event or at least 1 network request?

Reply:
There is no way unfortunately